src/commands/wizard/lwcGenerationCommand.ts

```diff
@@ -8,6 +18,70 @@ export class LwcGenerationCommand {
         this.extensionUri = extensionUri;
     }
 
+    static readFileAsJsonObject(
+        filePath: string,
+        callback: (error: Error | null, data: any) => void
+    ): void {
+        try {
+            const data = fs.readFileSync(filePath, { encoding: 'utf-8' });
+            const jsonObject = JSON.parse(data);
+            callback(null, jsonObject);
+        } catch (error) {
+            callback(error as Error, null);
+        }
+    }
+
+    static async getLwcGenerationPageStatus(): Promise<LwcGenerationCommandStatus> {
+        return new Promise<LwcGenerationCommandStatus>(async (resolve) => {
+            let landingPageExists = true;
+
+            const staticResourcesPath =
+                await TemplateChooserCommand.getStaticResourcesDir();
+            const landingPageJson = 'landing_page.json';
+            const landingPagePath = path.join(
+                staticResourcesPath,
+                landingPageJson
+            );
+
+            const lwcGenerationCommandStatus: LwcGenerationCommandStatus = {
+                sobjects: []
+            };
+
+            try {
+                await access(landingPagePath);
+            } catch (err) {
+                console.warn(
+                    `File '${landingPageJson}' does not exist at '${staticResourcesPath}'.`
+                );
+                landingPageExists = false;
+                lwcGenerationCommandStatus.error = (err as Error).message;
+            }
+
+            if (landingPageExists) {
+                this.readFileAsJsonObject(
+                    landingPagePath,
+                    (error: Error | null, data: any) => {
+                        if (error) {
+                            console.warn(`Error reading ${landingPageJson}`);
+                            lwcGenerationCommandStatus.error = (
+                                error as Error
+                            ).message;
+                        } else {
+                            lwcGenerationCommandStatus.sobjects =
+                                UEMParser.findFieldValues(
```

Comment:
I am wondering if we want to find _all_ "objectApiName" keys in the entire json, or just for the mcf/list, mcf/timedList, and mcf/genericLists? 

Because the purpose of generating the LWCs for view/edit/create is that the lists are the jumping point for sobjects. Looks like the only place objectApiName appears in the templates is under either `mcf/list` or `mcf/timedList` so this should still work.

Reply:
+1. I'd prefer an approach where, rather than blindly searching the landing page, we go the other way and get values based on known paths to those keys.